src/model/cloud-runner/k8s/kubernetes-logging.ts

```diff
@@ -0,0 +1,71 @@
+import { CoreV1Api, KubeConfig, Log } from '@kubernetes/client-node';
+import { Writable } from 'stream';
+import CloudRunnerLogger from '../services/cloud-runner-logger';
+import { CloudRunnerState } from '../state/cloud-runner-state';
+import fs from 'fs';
+import { CloudRunnerStatics } from '../cloud-runner-statics';
+class KubernetesLogging {
```

Comment:
Prefer leaving a new line between imports and classes, which is relevant for detecting diffs better.

Reply:
@webbertakken is it possible to setup a linting rule?